humble/distribution.yaml

```diff
@@ -5040,6 +5040,18 @@ repositories:
       url: https://github.com/KumarRobotics/ublox.git
       version: ros2
     status: maintained
+  ublox_dgnss:
+    release:
```

Comment:
consider adding a doc and a source entry

Reply:
should it be this https://github.com/aussierobots/ublox_dgnss

Reply:
@ivanpauno created a new pull request. Closing this one